tests/materialize/materialize-bigquery/snapshot.json

```diff
@@ -400,7 +400,7 @@
 ]
 {
   "applied": {
-    "actionDescription": "DROP TABLE IF EXISTS `estuary-sandbox`.whb_other_region.simple;\nDROP TABLE IF EXISTS `estuary-sandbox`.whb_other_region.duplicate_keys;\nDROP TABLE IF EXISTS `estuary-sandbox`.whb_other_region.multiple_types;\nDROP TABLE IF EXISTS `estuary-sandbox`.whb_other_region.formatted_strings;\nCREATE TABLE IF NOT EXISTS `estuary-sandbox`.whb_other_region.flow_materializations_v2 (\n\t\tmaterialization STRING NOT NULL,\n\t\tversion STRING NOT NULL,\n\t\tspec STRING NOT NULL\n)\nCLUSTER BY materialization;\n\nCREATE TABLE IF NOT EXISTS `estuary-sandbox`.whb_other_region.flow_checkpoints_v1 (\n\t\tmaterialization STRING NOT NULL,\n\t\tkey_begin INT64 NOT NULL,\n\t\tkey_end INT64 NOT NULL,\n\t\tfence INT64 NOT NULL,\n\t\tcheckpoint STRING NOT NULL\n)\nCLUSTER BY materialization, key_begin, key_end;\n\nUPDATE `estuary-sandbox`.whb_other_region.flow_materializations_v2 SET version = 'test', spec = '(a-base64-encoded-value)' WHERE materialization = 'tests/materialize-bigquery/materialize';"
+    "actionDescription": "\nCREATE TABLE IF NOT EXISTS `estuary-sandbox`.whb_other_region.flow_materializations_v2 (\n\t\tmaterialization STRING NOT NULL,\n\t\tversion STRING NOT NULL,\n\t\tspec STRING NOT NULL\n)\nCLUSTER BY materialization;\n\nCREATE TABLE IF NOT EXISTS `estuary-sandbox`.whb_other_region.flow_checkpoints_v1 (\n\t\tmaterialization STRING NOT NULL,\n\t\tkey_begin INT64 NOT NULL,\n\t\tkey_end INT64 NOT NULL,\n\t\tfence INT64 NOT NULL,\n\t\tcheckpoint STRING NOT NULL\n)\nCLUSTER BY materialization, key_begin, key_end;\n\nUPDATE `estuary-sandbox`.whb_other_region.flow_materializations_v2 SET version = 'test', spec = '(a-base64-encoded-value)' WHERE materialization = 'tests/materialize-bigquery/materialize';"
```

Comment:
Not something for this PR, but seeing it in the diff reminded me that we should move to using a more stable dataset for bigquery tests than `whb_other_region` - I actually deleted that one recently (not knowing it was used for tests) so I doubt these tests will work anymore. With the new materialization testing system we should be able to run bigquery tests in CI. Previously I held off on adding them to run automatically since there was an inconsistent/long delay in how long it would take the data to be available in bigquery to to validate the materialization results, but now the tests are driven differently and it shouldn't be any issue. So if/when we set up the tests to run automatically it would make sense to establish an "official" dataset for the tests to run in I think.